<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Superlattices</class>
 <widget class="QDialog" name="Superlattices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Superlattices</string>
  </property>
  <widget class="QGroupBox" name="SL_Box">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>541</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Target</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Target 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Target 2</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Setting</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>32</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sweep</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Shot</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>12</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="T1_sweep">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="T1_shot">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>48</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="T1_position">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Reprate</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="T1_reprate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Setting</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>32</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sweep</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Shot</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="T2_sweep">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="T2_shot">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>48</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="T2_position">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>8</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Reprate</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="T2_reprate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Repeat</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Repeat">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Repeat</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Repeat_str">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="SL_log">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>256</width>
      <height>141</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="SL_Start">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SL_Stop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
